config/routing.go

```diff
@@ -22,10 +22,7 @@ type Routing struct {
 
 type Router struct {
 
-	// Currenly supported Types are "reframe", "dht", "parallel", "sequential".
-	// Reframe type allows to add other resolvers using the Reframe spec:
-	// https://github.com/ipfs/specs/tree/main/reframe
-	// In the future we will support "dht" and other Types here.
+	// Currenly supported Types are "http", "dht", "parallel", "sequential".
```

Comment:
What is the source of truth here?  Is it https://github.com/ipfs/kubo/blob/master/docs/config.md#routingrouters-type or is a constant in code?  I think it would be ideal to link to the source of truth.

Reply:
the source of truth is `RouterType` , the type of the variable.